engine/src/flutter/impeller/entity/contents/content_context.h

```diff
@@ -92,18 +98,36 @@ using SolidFillPipeline =
 using RadialGradientFillPipeline =
```

Comment:
I think its about time to macro up this file....

Reply:
Ugh yea.  We might be able to get rid of a bit with macros.  I'll poke around a bit in a different pr.

Reply:
I have a draft PR that cuts the size of this file in about half: https://github.com/flutter/flutter/blob/664ac83be9185d9b3773a46549bb40b6a71cb1c2/engine/src/flutter/impeller/entity/contents/content_context.h

We could do a bit more to clean it up but that's probably good for now.